theia-go-docker/next.package.json

```diff
@@ -1,22 +1,22 @@
 {
     "private": true,
     "dependencies": {
+        "@theia/plugin-ext-vscode": "next",
         "@theia/editor-preview": "next",
-        "@theia/go": "next",
-        "@theia/messages": "next",
-        "@theia/navigator": "next",
-        "@theia/terminal": "next",
-        "@theia/outline-view": "next",
-        "@theia/preferences": "next",
         "@theia/getting-started": "next",
-        "@theia/git": "next",
-        "@theia/file-search": "next",
-        "@theia/markers": "next",
-        "@theia/search-in-workspace": "next",
         "@theia/json": "next",
         "@theia/textmate-grammars": "next"
     },
     "devDependencies": {
-        "@theia/cli": "next"
+        "@theia/cli": "next",
+        "@theia/debug": "next"
+    },
+    "scripts": {
+        "postinstall": "download-debug-adapters"
+    },
+    "adapterDir": "plugins",
+    "adapters": {
+        "vscode-builtin-go": "https://registry.npmjs.org/@theia/vscode-builtin-go/-/vscode-builtin-go-0.1.0.tgz",
```

Comment:
should be removed after https://github.com/theia-ide/theia/pull/4547 gets merged